.gitignore

```diff
@@ -13,4 +13,5 @@ nbproject
 /sites/default/_settings.php
 /profiles/openideal/sublime-openideal.sublime-project
 /profiles/openideal/sublime-openideal.sublime-workspace
-/.gitk-tmp.11456/
\ No newline at end of file
+/.gitk-tmp.11456/
```

Comment:
@israelLinnovate @nevos Can you make this more generic? (for example `.gitk-tmp*`)

Reply:
although it's not at all relevant to the PR..